ironic/drivers/modules/drac/raid.py

```diff
@@ -294,6 +294,44 @@ def clear_foreign_config(node, raid_controller):
         raise exception.DracOperationError(error=exc)
 
 
+def change_physical_disk_state(node, mode,
+                               controllers_to_physical_disk_ids=None):
```

Comment:
To simplify things a little, let's remove the controllers_to_physical-disk_ids on the above line and in the comment below since we always pass None

Reply:
Removed.